README.md

```diff
@@ -5,9 +5,21 @@ A discord bot created for use in a personal discord server.
 - TypeScript
 - Node.JS v14
 
-# Setup
-This setup is only for macOS/Ubuntu users at the moment.
+# macOS/Ubuntu Setup
 
 - Install TypeScript globally: `npm install typescript -g`
-- Install nodemon globally: `npm install nodemon -g`
-- Install packages: `npm install`
\ No newline at end of file
+- Install packages: `npm install`
+
+# Windows Setup 
```

Comment:
Now that I think about it, these new setups are done by all of us on all our environments so  I don't think we need to specify and environment setup in here.

Reply:
I think it still might be good to just have it to so anyone new who doesn't have it set up can just follow it easily.